### PR TITLE
Fix master branch

### PR DESCRIPTION
**Pull request by ikalnytskyi**
The intention of this PR is to ensure that latest `master` can pass CI.

* It bumps versions in `Cargo.lock` because some of them contain CVEs and some others use removed APIs.
* It fixes some lint errors.
* It adds new required parameter to gabbi test generation function.

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -106,7 +106,7 @@ jobs:
       - uses: actions-rs/cargo@v1
         with:
           command: rustdoc
-          args: -- -D intra-doc-link-resolution-failure
+          args: -- -D rustdoc::broken_intra_doc_links
```
Comment (ikalnytskyi) [inline .github/workflows/tests.yml:109]: **n.b.** It has been renamed.

File: src/storage/sql/models.rs [hunk 2]
```diff
@@ -96,10 +96,10 @@ pub fn combine_rows(
                 snippet,
                 changesets_by_snippet_id
                     .remove(&snippet_id)
-                    .unwrap_or_else(Vec::new),
+                    .unwrap_or_default(),
```
Comment (ikalnytskyi) [inline src/storage/sql/models.rs:99]: **n.b.** The linter asked for the change.

File: src/web/auth.rs [hunk 3]
```diff
@@ -9,15 +9,15 @@ use serde::Deserialize;
 
 pub use jwt::JwtValidator;
 
-#[derive(Debug, PartialEq, Deserialize)]
+#[derive(Debug, PartialEq, Eq, Deserialize)]
```
Comment (ikalnytskyi) [inline src/web/auth.rs:12]: **n.b.** The linter asked for the change.

**Review by malor — APPROVED** [review]
Thank you! 👍

**Pull request merged**